libs/net/ches/ChesService.cs

```diff
@@ -42,11 +44,12 @@ public class ChesService : IChesService
         /// <param name="client"></param>
         /// <param name="tokenHandler"></param>
         /// <param name="logger"></param>
-        public ChesService(IOptions<ChesOptions> options, ClaimsPrincipal user, OpenIdConnectRequestClient client, ILogger<IChesService> logger)
+        public ChesService(IOptions<ChesOptions> options, ClaimsPrincipal user, IHttpRequestClient client, JwtSecurityTokenHandler tokenHandler, ILogger<IChesService> logger)
```

Comment:
@jdtoombs I don't think you ever got this working.  I had to make a number of changes to get it working.